src/test/java/ParserTest.java

```diff
@@ -0,0 +1,46 @@
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class ParserTest {
+    @Test
+    public void getIndexTest() {
+        Parser p = new Parser("done 2");
```

Comment:
in larger tests, consider moving these kinda setup steps to separate methods. try to find out what do @Before and @After do in JUnit.